### PR TITLE
Prepare v0.2.3 release and modify CHANGELOG.md

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.3] - 2020-07-22
```
Comment (mariux) [inline CHANGELOG.md:9]: link ref is missing at end of file for `[0.2.3]`

Comment (soerenmartius) [reply CHANGELOG.md:9]: fixed in 523bf38

Comment (soerenmartius) [reply CHANGELOG.md:9]: resolved in 523bf38

File: CHANGELOG.md [hunk 2]
```diff
@@ -64,7 +70,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Implement support for `aws_route53_record` resource.
 
 [Unreleased]: https://github.com/mineiros-io/terraform-aws-route53/compare/v0.2.1...HEAD
```
Comment (mariux) [inline CHANGELOG.md:72]: 0.2.2

Comment (soerenmartius) [reply CHANGELOG.md]: resolved in f158812

**Pull request merged**